shoop/addons/admin_module/views/reload.py

```diff
@@ -56,4 +64,8 @@ def form_valid(self, form):
     def get(self, request, *args, **kwargs):
         if request.GET.get("ping"):
             return JsonResponse({"pong": time.time()})
+        elif request.GET.get("migrate"):
+            return JsonResponse({"message": run_command("migrate")})
+        elif request.GET.get("collectstatic"):
+            return JsonResponse({"message": run_command("collectstatic")})
```

Comment:
Just what my plan was initially.

Well we have to reload the application twice. Before and after migrations and collectstatic. Currently the reload request does not return anything. That HttpResponse return is never reached because of reload. So doing migrations after reload seems impossible.

Also this way we offer shop admin more detailed information about what is done while reloading (like steps and log). All of the steps can be performed separately so in this way shop admin needs to just reload again if errors occur. Also I think it's cool to start migrating and doing collectstatic ONLY after the application is up again after initial reload. As I understood the reload itself might take while.

If we actually managed to do everythng in one request still debugging, would be harder in error cases.